stylesheet/styles.css

```diff
@@ -0,0 +1,276 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0px;
+  padding: 0px;
+}
+/* Header */
+header {
+  background-color: black;
+  color: white;
+  background-image: url('https://boone824.github.io/adopt-a-dog-winter2020/images/header-image.jpg');
+}
+#brand-container {
+  display: flex;
+  justify-content: space-between;
+  width: 100%
+
+}
+#checkout-btn {
+  background-color: green;
+  padding: 5px 20px;
+  color: white;
+  margin-right: 2em;
+  width: 120px;
+  height: 25px;
+}
+#checkout-btn:hover {
+  background-color: black;
+  cursor: pointer;
+  transition: all 0.5s ease;
+  color: red;
+  border: 1px red solid;
+
+}
+#brand-logo {
+  margin-top: 2em;
+  margin-left: 4em;
+}
+#brand-checkout p {
+  text-align: right;
+  margin-right: 2em;
+  margin-top: 2em;
+}
+ul {
+  margin: 0;
+  padding: 0;
+
+}
+nav a {
+  text-decoration: none;
+  font-size: 16px;
+  color: black;
+  line-height: 2em;
+}
+nav a:hover {
+  text-decoration: underline;
+}
+nav li {
+  list-style: none;
+  float: left;
+  display: inline;
+  padding: 0px 20px;
+}
+nav {
+  font-size: 1rem;
+  background-color: #e7e7e7;
+  height: 3rem;
+  margin-top: 12rem;
+}
+nav ul, {
+  margin: 0 auto;
+  width: 70em;
+
+}
+main {
+  width: 80%;
```

Comment:
I'd say width should be 100%, but set a max width so it doesn't go too wide on desktop